stable/mysql/templates/secrets.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if not .Values.existingSecret }}
+{{ if or (not .Values.allowEmptyRootPassword) (not .Values.allowEmptyPassword) }}
```

Comment:
`(!a | !b) = (a & b)`? :)
Not sure you need this block though.

Reply:
you catch me :)